package.json

```diff
@@ -31,9 +31,9 @@
     "dependencies": {
         "@emotion/react": "^11.11.3",
         "@emotion/styled": "^11.11.0",
-        "ag-grid-community": "31.1.1",
-        "ag-grid-enterprise": "31.1.1",
-        "ag-grid-react": "31.1.1",
+        "ag-grid-community": "31.2.1",
+        "ag-grid-enterprise": "31.2.1",
+        "ag-grid-react": "31.2.1",
```

Comment:
@BSd3v make sure to commit changes to `package-lock.json` here

Reply:
AG Grid and our package version should be insync, thus its easier to know what version dag is pointing to.